.travis.yml

```diff
@@ -3,13 +3,15 @@ language: php
 sudo: false
 
 php:
+  - 5.5
   - 5.6
   - 7.0
+  - 7.1
   - hhvm
 
 matrix:
   include:
-    - php: 5.6
+    - php: 5.5
```

Comment:
5.5 for this is failing (and the tests are `--quiet`?? so we can't see the output)